### PR TITLE
pvlib-python v0.7.0

### PR DESCRIPTION
**Pull request by CameronTStark**
Checklist
* [X] Used a fork of the feedstock to propose changes
* [X] Reset the build number to `0` (if the version changed)
* [x] Ensured the license file is being packaged.
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)

Updates version to v0.7.0, updated to current numpy and pandas version requirements and included myself as a recipe-maintainer.

The link in the third bullet above did not work an there wasn't a reference to "conda-smithy" in the [pvlib Github wiki](https://github.com/pvlib/pvlib-python/wiki/Release-procedures#update-conda-forge-packages) so I'm not sure if this is still applicable.

### CODE REVIEW

**Comment by CameronTStark** [comment]
@conda-forge-admin, please rerender

**Comment by CameronTStark** [comment]
@wholmgren the builder seems like it's trying to build for 2.7 which is causing the error. I looked in various files for a command I could alter but I couldn't find any. Do you have an idea where this may be?

**Comment by wholmgren** [comment]
No, sorry, I don't know how to do that. A little bit of googling and reading conda build and conda forge docs didn't provide a clear answer.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,8 +19,8 @@ requirements:
     - pip
     - python
```
Comment (wholmgren) [inline recipe/meta.yaml]: @CameronTStark maybe it's as simple as changing this to `python >=3.5`? Might need to modify the `run` section below too.

Comment (CameronTStark) [reply recipe/meta.yaml]: That was it, thanks!

**Comment by wholmgren** [comment]
All checks passed and that's all that matters to me. @CameronTStark do you have ability to merge this?

**Comment by CameronTStark** [comment]
I don't.  The merge message says:

This branch has no conflicts with the base branch
Only those with write access to this repository can merge pull requests.

**Comment by wholmgren** [comment]
of course I can't figure out how to add you. I'll merge this time and maybe we can figure it out next time.

**Pull request merged**